aspnetcore/security/enforcing-ssl.md

```diff
@@ -41,38 +43,43 @@ The following code calls `UseHttpsRedirection` in the `Startup` class:
 
 The preceding highlighted code:
 
-* Uses the default [HttpsRedirectionOptions.RedirectStatusCode](/dotnet/api/microsoft.aspnetcore.httpspolicy.httpsredirectionoptions.redirectstatuscode) (`Status307TemporaryRedirect`).
+* Uses the default [HttpsRedirectionOptions.RedirectStatusCode](/dotnet/api/microsoft.aspnetcore.httpspolicy.httpsredirectionoptions.redirectstatuscode) ([Status307TemporaryRedirect](/dotnet/api/microsoft.aspnetcore.http.statuscodes.status307temporaryredirect)).
```

Comment:
Prefer the xref style for the API link, since you're adding a new one.

Reply:
Holding for a future PR. The whole topic is full of these, and I'm in a bit of a 🏃.